src/sciline/visualize.py

```diff
@@ -83,18 +83,16 @@ def to_graphviz(
 
 
 def _to_subgraphs(graph: FormattedGraph) -> Dict[str, FormattedGraph]:
-    def get_subgraph_name(name: str) -> str:
-        sgname = name.split('[')[0]
-        if sgname == 'Series':
+    def get_subgraph_name(name: str, kind: str) -> str:
+        if kind == 'series':
             # Example: Series[RowId, Material[Country]] -> RowId, Material[Country]
             return name.partition('[')[-1].rpartition(']')[0]
-        return sgname
```

Comment:
This used to return the first name after splitting with `[`, but is now returning the whole thing?
Is that intentional?

Reply:
No, it's an oversight. It means that before all subgraphs that return `Series[*]` were merged. Is this intentional?